pyproject.toml

```diff
@@ -76,6 +76,7 @@ doc = [
     "scanpydoc>=0.7.7",
     "typing_extensions; python_version < '3.8'",
     "zarr",
+    "awkward>=1.9.0rc12,<2",
```

Comment:
```suggestion
    "awkward>=1.9.0,<2",
```

Reply:
we may even need 1.10rcXX to make all the tests pass? e.g. https://github.com/scikit-hep/awkward/pull/1674 is not in the 1.9 release yet.

Reply:
```suggestion
    "awkward>=1.10.0rc2,<2",
```